external/dockerfile_functions.sh

```diff
@@ -397,8 +397,8 @@ print_criu_install() {
                 "\n\t&& git clone https://github.com/ibmruntimes/criu.git \\" \
                 "\n\t&& cd criu \\" \
                 "\n\t&& git fetch origin \\" \
-                "\n\t&& git reset --hard origin/0.43.0-release \\" \
-                "\n\t&& make PREFIX=/usr install \\" \
+                "\n\t&& git reset --hard origin/0.44.0-release \\" \
+                "\n\t&& make PREFIX=/usr CONFIG_AMDGPU=n install \\" \
```

Comment:
Is that always true?  That there's no AMDGPU involved.

Reply:
Currently the AMD GPU module built has some bugs, so disable it for now.

Reply:
Assuming this conversation can be resolved.